lib/lrama/grammar.rb

```diff
@@ -400,6 +408,10 @@ def normalize_rules
           add_nterm(id: rule._lhs)
         end
       end
+
+      if @parameterizing_resolver.term
+        add_term(id: @parameterizing_resolver.term)
```

Comment:
I'm wondering this part, because
1. Only non terminals (nterm) are newly generated by applying parameterizing rules. In general terminals are defined by `%token` and non terminals are defined as LHS of rules. All of arguments are defined by `%token` as rules then LHS of newly generated rules are only new symbols we need to take care about here.
2. Non terminals we need to take care about is not always single. If 2 or more applications happen, 2 or more non terminals might be generated (exception is when all arguments of applications are same).

In short, this codes will be similar with https://github.com/ruby/lrama/pull/285/files#diff-bc2fdb878ce319ff339914b979ff298e61490736ab36ee1080bd8f0ff85b4a87R402-R405.

Reply:
I guess these codes are not needed. Nterm for new rules are covered by https://github.com/ruby/lrama/pull/285/files#diff-bc2fdb878ce319ff339914b979ff298e61490736ab36ee1080bd8f0ff85b4a87R402-R405.

Reply:
This is a needless process. It was caused by the same thing as https://github.com/ruby/lrama/pull/285#discussion_r1421240833. It is no longer needed and was removed in the following commit:
https://github.com/ruby/lrama/pull/285/commits/e8f022787f79e76fd8b2f0fdbbd10645fa6917a0